src/mixins/_button.scss

```diff
@@ -89,6 +91,62 @@
     transform: none;
   }
 
+  .c-button__extra {
+    display: flex;
+    font-size: ms.step(1);
+    margin: 0 ms.step(-4);
```

Comment:
Can we create a design token for `ms.step(-4`)? Maybe `size.$spacing-gap-button-extra`?

Reply:
Thanks, @tylersticka! Should I also create a design token for `ms.step(1)`?

Reply:
@gerardo-rodriguez That's probably a good idea, but I didn't have a clear suggestion on what it should be called so I decided not to suggest it. 😆 

Maybe we need to create a set of icon size tokens, with the default being modular scale step 0 and `medium` or `large` or something for this? I'm open to suggestions.

Reply:
LOL Got it, thanks! I'll think a bit more about this! 😉

Reply:
> Can we create a design token for `ms.step(-4`)? Maybe `size.$spacing-gap-button-extra`?

Done! https://github.com/cloudfour/cloudfour.com-patterns/pull/1298/commits/5fcb4fa6baffe2d3e174051703522456391e7042

Reply:
> Maybe we need to create a set of icon size tokens, with the default being modular scale step 0 and `medium` or `large` or something for this? I'm open to suggestions.

Done! https://github.com/cloudfour/cloudfour.com-patterns/pull/1298/commits/884cb0a7c198322b017029b8b96ee21d6bd279cd